timeside/server/management/commands/timeside-celery-worker.py

```diff
@@ -13,8 +13,7 @@ def restart_celery(*args, **kwargs):
     kwargs = kwargs['kwargs']
     kill_worker_cmd = 'pkill -9 celery'
     subprocess.call(shlex.split(kill_worker_cmd))
-    start_worker_cmd = 'celery -A worker worker --loglevel=%s --logfile=%s --uid=%uid --gid=%gid' %
-        (kwargs['loglevel'], kwargs['logfile'], kwargs['uid'], kwargs['gid'])
+    start_worker_cmd = 'celery -A worker worker --loglevel=%s --logfile=%s --uid=%s --gid=%s' % (kwargs['loglevel'], kwargs['logfile'], kwargs['uid'], kwargs['gid'])
```

Comment:
Can we share the celery parameters with `worker.sh` to avoid duplicate celery parameters in multiple files ?
See: https://github.com/Parisson/TimeSide/issues/180#issuecomment-614082312

> I can see two solutions:
> - Create a variable like celery_args that would be used by timeside-celery-worker.py and worker.sh
> - Refactor timeside-celery-worker.py to call worker.sh

Reply:
I agree with the 1st solution